README.md

```diff
@@ -561,6 +561,25 @@ def autocorrect(_processed_source, offense)
 end
 ```
 
+## Rake
+
+You can add Rake support by adding the following to your `Rakefile`:
+
+    require "erb_lint/rake_task"
+    ERBLint::RakeTask.new
+
+The Rake task can be customized as follows:
+
+    # Custom arguments.
+    ERBLint::RakeTask.new { %w[--autocorrect .] }
+
+    # Custom name and arguments.
+    ERBLint::RakeTask.new(:erb_auto_correct) { %w[--autocorrect .] }
```

Comment:
I'm not super convinced about passing arguments as the value yielded by the block. What do you think about this?
```suggestion
    ERBLint::RakeTask.new(:erb_auto_correct, args: %w[--autocorrect .])
```